js/src/core/app.js

```diff
@@ -32,7 +32,8 @@ Wu.App = Wu.Class.extend({
 	},
 
 	_raven : function () {
-		if(window.testMode) return true;
+		if(window.testMode || location.hostname === "localhost") return true;
```

Comment:
Isn't there another way to do this? I mean, to get the logs AND Raven on localhost? 

If not, this is a good workaround I guess.